examples/transformation/test_transformation.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022-2023 ImpactX contributors
+# Authors: Ryan Sandberg, Axel Huebl, Chad Mitchell
+# License: BSD-3-Clause-LBNL
+#
+# -*- coding: utf-8 -*-
+
+import numpy as np
+
+import amrex
+from impactx import (  # NOQA
+    Config,
+    ImpactX,
+    ImpactXParIter,
+    RefPart,
+    TransformationDirection,
+    coordinate_transformation,
+    distribution,
+    elements,
+)
+
+
+def get_particle_data(pc):
+    for lvl in range(pc.finest_level + 1):
```

Comment:
Please add a doc string
```suggestion
    for lvl in range(pc.finest_level + 1):
        """Doc string here
        
        Does this copy the data?
        """
```